Source/MIKMIDIDeviceManager.h

```diff
@@ -138,6 +139,14 @@ extern NSString * const MIKMIDIEndpointKey;
 - (BOOL)sendCommands:(MIKArrayOf(MIKMIDICommand *) *)commands toVirtualEndpoint:(MIKMIDIClientSourceEndpoint *)endpoint error:(NSError **)error;
 
 
+/**
+ *  The current MIDI output port.
+ *  Typically this should only be required for custom Core MIDI implementations in C/C++ where MIKMIDI has already been implemented to handle endpoint setup.
+ *  Creates a new MIDI output port if one does not already exist.
+ */
+//- (MIKMIDIOutputPort *)outputPort;
```

Comment:
Looks great! Do you mind pulling out the commented out method declaration here? After that change, I'll go ahead and merge.